### ISSUE TITLE
story: executing job

### ISSUE THREAD

**Issue by yashipro13** [gopay-bootcamp/octavius#9]
#### Description
Client should have an accessibility to execute jobs that have already been created by running command `job execute <id>`. On running the given command, given the proper config file, the job should be executed and the expected stream should be obtained at the client end.

#### Present Behaviour
No such behaviour

#### Expected Behaviour 
Client should be able to execute job from his end by running `job execute <id>`

#### Screenshots
Not applicable in this case.
